Client/Frontend/Browser/Search/SearchEngines.swift

```diff
@@ -62,9 +62,14 @@ class SearchEngines {
         self.orderedEngines = getOrderedEngines()
     }
     
-    func setupDefaultRegionalSearchEngines() {
-        guard let region = Locale.current.regionCode,
-            let searchEngine = SearchEngines.defaultRegionSearchEngines[region] else { return }
+    func regionalSearchEngineSetup(for locale: Locale = Locale.current) {
```

Comment:
This isn't needed, as we never pass in `locale`, so maybe just use `setupDefaultRegionalSearchEngines` and internalize the local parameter?

Reply:
I can remove it but this is good if we would want to add unit tests for it, much easier to test regional url params

Reply:
It's easier to add unit tests with this param